ts/input/tex/ams/AmsMethods.ts

```diff
@@ -233,6 +234,111 @@ AmsMethods.HandleOperatorName = function(parser: TexParser, name: string) {
   parser.i = 0;
 };
 
+/**
+ * Handle sideset.
+ * @param {TexParser} parser The calling parser.
+ * @param {string} name The macro name.
+ */
+AmsMethods.SideSet = function (parser: TexParser, name: string) {
+  //
+  //  Get the pre- and post-scripts, and any extra material from the arguments
+  //
+  const [preScripts, preRest] = splitSideSet(parser.ParseArg(name));
+  const [postScripts, postRest] = splitSideSet(parser.ParseArg(name));
+  const base = parser.ParseArg(name);
+  let mml = base;
+  //
+  //  If there are pre-scripts...
+  //
+  if (preScripts) {
+    //
+    //  If there is other material...
+    //
+    if (preRest) {
+      //
+      //  Replace the empty base of the prescripts with a phantom element of the
+      //    original base, with width 0 (so of the correct height and depth).
```

Comment:
Missing word?

Reply:
Not really, but perhaps could be better worded.  The phantom element is "of the correct height and depth".  I'll reword.